project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1366 @@
+
+references:
+
+    options:
+    #### 1.0 Common Options For Heat Pump, 12 options ####
+      - &1_heating_fuel
+        option: Heating Fuel|Electricity
+        apply_logic:
+          - not: Heating Fuel|None
+
+      - &2_cooling_efficiency_ducted
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+        apply_logic: &logic_ducted_hvac_no_ashp
+          - HVAC Has Ducts|Yes
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity ASHP
+      
+      - &3_cooling_efficiency_ductless
+        option: HVAC Cooling Efficiency|Non-Ducted Heat Pump
+        apply_logic: &logic_ductless_hvac_no_mshp
+          - HVAC Has Ducts|No
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity MSHP
+
+      - &4_100_percent_conditioned
+        option: HVAC Cooling Partial Space Conditioning|100% Conditioned
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+      
+      - &5_cooling_setpoint_no_offset
+        option: Cooling Setpoint Has Offset|No
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+
+      - &6_cooling_setpoint_offset_magnitude_0
+        option: Cooling Setpoint Offset Magnitude|0F
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+      
+      - &7_cooling_setpoint_offset_period_none
+        option: Cooling Setpoint Offset Period|None
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+
+      - &8_heating_setpoint_no_offset
+        option: Heating Setpoint Has Offset|No
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+
+      - &9_heating_setpoint_offset_magnitude_0
+        option: Heating Setpoint Offset Magnitude|0F
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+      
+      - &10_heating_setpoint_offset_period_none
+        option: Heating Setpoint Offset Period|None
+        apply_logic:
+          - or:
+            - *logic_ducted_hvac_no_ashp
+            - *logic_ductless_hvac_no_mshp
+
+      - &11_heating_type_and_fuel_ashp 
+        option: HVAC Heating Type And Fuel|Electricity ASHP
+        apply_logic:
+          - HVAC Has Ducts|Yes
+          - not: Heating Fuel|None
+
+      - &12_heating_type_and_fuel_mshp 
+        option: HVAC Heating Type And Fuel|Electricity MSHP
+        apply_logic:
+          - HVAC Has Ducts|No
+          - not: Heating Fuel|None
+
+    #### 1.1 Electric Resistance Heating, 10 options ####
+      - &1_electric_resistance_heating_heating_fuel
+        option: Heating Fuel|Electricity
+        apply_logic:
+          - not: Heating Fuel|None
```

Comment:
```
not:
  or:
    - Heating Fuel|None
    - Heating Fuel|Electricity
```

Reply:
revised

Reply:
Not yet I think, also, this is the same as *1_heating_fuel, you could just use that.

Reply:
done.